Applications/PlazaDetails/ExecuteService.qmd

```diff
@@ -4,34 +4,34 @@ aliases:
   - '/Applications/PlazaDetails/ExecuteService.html'
 ---
 
-<div style="text-align: justify">
 
-Services can be executed using the tools provided by openEO. The table below provides a brief summary of the different tools available:
+Users can use the services offered in the openEO Algorithm Plaza to analyse their data. These services consist of openEO-based User Defined Processes (UDP). Here is a concise overview of the tools used to implement and use these available services:
+
 
 | Tool | Description | 
 | --- | --- |
 | openEO Web Editor | A web-based interface for executing openEO processes. | 
 | Command Line Interface (CLI)	 | Allows execution of openEO processes through the command line. | 
-| Python Client | 	A Python library facilitating interaction with openEO services programmatically. | 
+| Python Client | 	A Python library, facilitating programmatic interaction with openEO services. | 
 | R Client	 | An R library for integrating openEO services into R scripts. | 
 
-When publishing a service on the openEO algorithm plaza, a user can choose to provide the following information in the service details:
-
-* An executable link which redirects the user to the online orchestrator's user interface. If this is the case, an `ACCESS SERVICE` button will appear when opening an openEO algorithm plaza service.
 
-* Sample code in the service description on how to execute a service programmatically. 
+Users have the flexibility to use their preferred tool to execute the services. Each service has a `Get Started` button within the plaza. This button offers available execution options for the service, such as:
 
+* "Execute in Web Editor": Allows users to run the service in the online user interface.
+* "See client examples": Provides code examples for programmatically executing the service using Python, R, or JavaScript.
 
 ::: {.callout-important}
 
-There is a limitation when executing a service (User Defined Processes) in Copernicus Dataspace Ecosystem, that it only works collection from 2017 or so onwards. Moreover, it is recommended to test it for multiple consecutive years.
+A limitation when executing a service in the Copernicus Data Space Ecosystem is that it only works collection from 2017 or so onwards. Therefore, it is recommended to test it for multiple consecutive years
```

Comment:
reconsider this part of the sentence; the grammar is incorrect: it only works collection from 2017 or so onwards.

Reply:
is the root cause that we only have collections from 2017 onwards? Maybe its better to be specific and mention that instead